### ISSUE TITLE
[Could you please update it to support controlnet]?Thank you so much!

### ISSUE THREAD

**Issue by axonaxis** [xrpgame/xyz_plot_script#20]
Can you please update to support controlnet? I believe it is a revolutionary plug-in,Someone has now turned it into an extension of AUTO1111.For More detailed ,Here is the link https://github.com/lllyasviel/ControlNet;https://github.com/Mikubill/sd-webui-controlnet;
Thank you so much.

**Comment by Woisek** [comment]
Yeah, since the entries are already there, please make them functional. Thanks.
